packages/blade/src/components/BottomSheet/BottomSheet.web.tsx

```diff
@@ -297,7 +297,14 @@ const _BottomSheet = ({
         preventScrollingRef.current = newY < upperSnapPoint;
       }
 
-      const shouldClose = newY < lowerSnapPoint;
+      // This ensure that the lower snapPoint will always have atleast some buffer
+      // When the bottomsheet total height is less than the lower snapPoint
+      // Video walkthrough: https://www.loom.com/share/a9a8db7688d64194b13df8b3e25859ae
```

Comment:
Anything extra in video not covered in comments?

Reply:
No i think the comment covers the "why" part. Attached the video just for extra context since it's a bit hard to visualise  whats going on here.